src/editors/containers/ProblemEditor/components/EditProblemView/index.scss

```diff
@@ -2,4 +2,14 @@
   .editProblemView-settingsColumn {
     flex-basis: 320px;
   }
-}
\ No newline at end of file
+}
+
+// overrides paragon in order to make checked radio and checkboxes green
+$checked-color: '%230D7D4D';
```

Comment:
Can we open an issue or PR with paragon to fix this? This can be only a very temporary solution, because other themes have their own style and this will override them as well.